src/app/app.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component} from '@angular/core';
+import { CancellationService } from './cancel.service';
+ 
+@Component({
+    selector: 'my-app',
+    template: `<div>
```

Comment:
Also for template https://github.com/IvanYermilov/AngularRxJs/pull/1/files#r1118526509